package.json

```diff
@@ -19,28 +19,32 @@
     }
   },
   "scripts": {
-    "build": "tsc --build && lerna run build --scope codemirror-graphql --scope graphiql --scope graphql-language-service --scope graphql-language-service-server",
+    "build": "yarn run build-clean && yarn build-ts && yarn build-js",
+    "build-js": "lerna run build --scope codemirror-graphql --scope graphql-language-service-server --scope graphiql graphql-language-service",
+    "build-ts": "tsc --build",
+    "build-clean": "tsc --build --clean && rm -rf 'packages/**/src/{dist,esm,bundle}' && lerna run build-clean --parallel",
```

Comment:
I'm not comfortable with `dist` / `esm` / `bundle` being inside the `src` folder long-term. We should try and refactor that in a follow-up PR.